src/components/Reviews/ReviewCard.js

```diff
@@ -1,5 +1,11 @@
+import {useNavigate} from 'react-router-dom'
+
 export const ReviewCard = ({review}) => {
-    return <li className='reviewCard'>
+    const navigate = useNavigate()
+    const handleClick = () => {
+        navigate(`/reviews/${review.review_id}`)
+    }
+    return <li onClick={handleClick} className='reviewCard'>
```

Comment:
try using a React Router link here - if you click on of the cards in the browser the url doesn't change